pom.xml

```diff
@@ -15,127 +15,130 @@
-        <maven.compiler.target>11</maven.compiler.target>
-        <ehrbase.sdk.version>96f3d62</ehrbase.sdk.version>
-        <commons.lang3.version>3.8</commons.lang3.version>
-        <jacoco.version>0.8.6</jacoco.version>
-        <dependency-check.version>6.0.3</dependency-check.version>
-    </properties>
+  <properties>
+    <maven.compiler.release>11</maven.compiler.release>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <ehrbase.sdk.version>96f3d62</ehrbase.sdk.version>
+    <commons.lang3.version>3.8</commons.lang3.version>
+    <jacoco.version>0.8.6</jacoco.version>
+    <dependency-check.version>6.0.3</dependency-check.version>
+    <aql-editor-backend-starter.version>0.0.3</aql-editor-backend-starter.version>
+  </properties>
 
-    <!-- Spring Boot starter parent is taking over most parts of the dependency management -->
-    <parent>
+  <!-- Spring Boot starter parent is taking over most parts of the dependency management -->
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.1.RELEASE</version>
+    <relativePath/>
+    <!-- lookup parent from repository -->
+  </parent>
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-actuator</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.springfox</groupId>
+      <artifactId>springfox-boot-starter</artifactId>
+      <version>3.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>io.springfox</groupId>
+      <artifactId>springfox-swagger-ui</artifactId>
+      <version>3.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.ehrbase</groupId>
+      <artifactId>aql-editor-backend-spring-boot-starter</artifactId>
+      <version>${aql-editor-backend-starter.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.ehrbase.openEHR_SDK</groupId>
+      <artifactId>client</artifactId>
+      <version>${ehrbase.sdk.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.ehrbase.openEHR_SDK</groupId>
+      <artifactId>aql</artifactId>
+      <version>${ehrbase.sdk.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.ehrbase.openEHR_SDK</groupId>
+      <artifactId>test-data</artifactId>
+      <version>${ehrbase.sdk.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>${commons.lang3.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+    </dependency>
+  </dependencies>
```

Comment:
These dependencies should not be needed anymore

Reply:
true